debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -56,6 +58,16 @@
 @NotThreadSafe
 public class KafkaDatabaseHistory extends AbstractDatabaseHistory {
 
+    private static final String CLEANUP_POLICY_NAME = "cleanup.policy";
+    private static final String CLEANUP_POLICY_VALUE = "delete";
+    private static final String RETENTION_MS_NAME = "retention.ms";
+    private static final long RETENTION_MS_MAX = Long.MAX_VALUE;
+    private static final long RETENTION_MS_MIN = 157_680_000_000L; // 5 years
```

Comment:
Let's use `Duration.of(5 , ChronoUnit.YEARS)`, as per the general preference for using the semantical temporal types of Java 8.

Reply:
Done